lib/alex_koin/slack.ex

```diff
@@ -74,20 +74,17 @@ defmodule AlexKoin.SlackRtm do
       SlackCommands.transfer_coin(coin, user_wallet(user), user_wallet(to_user), memo)
 
       # Notify the recipient of the new koin
-      #notify_msg = "<@#{user.slack_id}> just transfered 1.0 :akc: with the memo: '#{memo}'"
-      #dm_channel = Slack.Web.Im.open(%{user: to_user.slack_id})
-      #  |> Map.get("channel")
-      #  |> Map.get("id")
-
-      #send_raw_message({notify_msg, nil}, dm_channel, slack)
+      notify_msg = "<@#{user.slack_id}> just transfered 1.0 :akc: with the memo: '#{memo}'"
```

Comment:
I'd kinda like to see these three lines off in their own method. `notify_receiver` or something? This is a side effect of transferring the coin, not really central to it. that said, you'd need a bunch of arguments (to_user, from_user, memo, and slack) so it's borderline. I'll leave it up to you.